modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -146,7 +146,7 @@ definition {
 
 			FDSCustomView.selectView(key_itemName = "TEST");
 
-			ObjectCustomViews.assertColumnNotPresent(columnName = "Description");
+			FDSCustomView.assertColumnNotPresent(columnName = "Description");
```

Comment:
These use different paths, can you check if this works correctly?

Reply:
Yes, they are different paths. I'll fix that.